src/peg/adql2.1.peg

```diff
@@ -482,10 +477,6 @@ user_defined_function <-
 	(_ value_expression (_ ',' _ value_expression)* _)?
 	')'
 
-numeric_primary	<-
-	value_expression_primary
-	/ numeric_value_function
```

Comment:
Hu?  How can this disapear without breaking the grammar?

Reply:
It was duplicated for some unknown reason:  [line 485])https://github.com/ivoa/lyonetia/blob/master/src/peg/adql2.1.peg#L485) and [line 558](https://github.com/ivoa/lyonetia/blob/master/src/peg/adql2.1.peg#L558).